server/public/styles.css

```diff
@@ -112,6 +112,10 @@ header {
   background-color: #3ea8a8;
 }
 
+.monthly-expense-list-table {
+  width: 90%;
+}
```

Comment:
I feel like this is not necessary. If you set the width of all tables to 100%, then you can make a container class that adds some padding on the left and right sides and just reuse that in each of your pages.